internal/manifest/v1beta1/custom_resource.go

```diff
@@ -73,7 +74,7 @@ func PreDeleteDeleteCR(
 		return ErrWaitingForAsyncCustomResourceDeletion
 	}
 
-	if !k8serrors.IsNotFound(err) {
+	if !k8serrors.IsNotFound(err) && !meta.IsNoMatchError(err) {
```

Comment:
if you ignore it here, it will be now thrown in line 86 on the get request, thus the error will not be fixed I believe. Could you confirm that?

Reply:
no, in line 86, the err is related to get manifest CR, but this err here is for the CR which defined in manifest spec.data